src/test/java/de/zalando/sprocwrapper/proxy/SubVirtualShardKeyStrategy.java

```diff
@@ -0,0 +1,10 @@
+package de.zalando.sprocwrapper.proxy;
+
+import de.zalando.sprocwrapper.sharding.VirtualShardKeyStrategy;
+
+/**
+ * @author Soroosh Sarabadani
+ */
+public class SubVirtualShardKeyStrategy extends VirtualShardKeyStrategy {
```

Comment:
Excuse me, i could not get you by moving to the test scope. You mean just moving it in test class as a static class?